### PR TITLE
Re-use upload gisaid log between bundles

### PR DESCRIPTION
**Pull request by Mropat**
## Description
Enable continuing to uplaod to gisaid when a batch is re-run

### Changed

- GISAID upload API will find and re-use the gisaid upload log file from older versions
- Update regex

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by mayabrandi — APPROVED** [review]
Looks clean to me! 👍

File: cg/constants/constants.py [hunk 1]
```diff
@@ -53,3 +53,5 @@ class DataDelivery(StrEnum):
 
 STATUS_OPTIONS = ("affected", "unaffected", "unknown")
 ANALYSIS_TYPES = ["tumor_wgs", "tumor_normal_wgs", "tumor_panel", "tumor_normal_panel"]
+
+SARS_COV_REGEX = "\(|\)|\d{2}CS\(|\)|\d{6}"
```
Comment (henningonsbring) [inline cg/constants/constants.py]: I dont undestand why all these escaping characters are used. To match the string we want I would In bash do: "^[0-9]{2}CS[0-9]{6}$". Here "^" indicates beginning and "$" indicates end. Now sure how to translate that into python.

File: cg/constants/constants.py [hunk 2]
```diff
@@ -53,3 +53,5 @@ class DataDelivery(StrEnum):
 
 STATUS_OPTIONS = ("affected", "unaffected", "unknown")
 ANALYSIS_TYPES = ["tumor_wgs", "tumor_normal_wgs", "tumor_panel", "tumor_normal_panel"]
+
+SARS_COV_REGEX = "^\d{2}CS\d{6}"
```
Comment (henningonsbring) [inline cg/constants/constants.py:57]: `SARS_COV_REGEX = "^\d{2}CS\d{6}$"`

Also match end of line? https://blog.finxter.com/python-regex-start-of-line-and-end-of-line/

**Review by henningonsbring — CHANGES_REQUESTED** [review]

File: cg/constants/constants.py (hunk 2 above)
Comment (Mropat) [reply cg/constants/constants.py:57]: done

**Pull request merged**